### ISSUE TITLE
Setup After Building. What to do next.

### ISSUE THREAD

**Issue by jamgro2434** [AlienFlight/Flight-Controllers#4]
I've already ordered all components and boards from OSHPark and mouser/digikey. Once I get the parts down, What would I need to do next to get cleanflight to upload changes to the board? I'm assuming theres a program or firmware that the boar needs to be running to be able to send and receive this info from their respective flight configurators. 

BTW: When ordering from OSHPark they send 3 boards so i'll have at least one extra if i dont mess up the other two. Would a $50 price tag be fair? That's comparable to other flight controllers around. This is supposed to be the Cadillac of boards so i figure that would raise its value to other boards. But then again I dont think I could provide any real support for it other than replacement components. (I would test them for functionality before sending it anywhere) But I'm just a guy doing the manual labor for some parts on a board. I don't know. Give me some feedback and tell me what I can do to keep it legal as well as fair and proper and what not.

**Comment by Franke123** [comment]
@jamgro2434 I'm gonna leave the first part to other people with more knowledge (Summoning @madman1412) But as for the second part I think $50 would be fair. Also you might consider ordering a stencil if you don't already have one I hear they make SMD a lot easier. [Here](https://www.oshstencils.com/) is a stencil site.

**Comment by jamgro2434** [comment]
yeah got one of those too. I figured it'd be better than me glopping on solder paste onto the pads

**Comment by Franke123** [comment]
@jamgro2434 Good luck on the build, keep us updated :)
[Here's](https://www.facebook.com/groups/AlienWii/) a link to the facebook group for AW, theres a ton of super useful and smart people there.

**Comment by Jtwalkz** [comment]
I have a file saved for F3 firmware flashing for the AF, but the link from
where I saved it from has been removed. I do not want to share it in case
it has been removed for a reason.
